config/config.yml

```diff
@@ -161,10 +161,20 @@ MAX_KUDOS_TO_SHOW: 50
 MAX_FAVORITE_TAGS: 20
 
 # Used by Rack Attack, periods are measured in seconds
-RATE_LIMIT_NUMBER: 300
-RATE_LIMIT_PERIOD: 300
-RATE_LIMIT_LOGIN_ATTEMPTS: 5
-RATE_LIMIT_LOGIN_PERIOD: 20
+RATE_LIMIT:
+        unicorn_elastic:
+                limit: 300
+                period: 300
+        unicorn_elastic_bookmarks:
+                limit: 300
+                period: 300
+        default:
```

Comment:
Are "default" and "login" values for `HTTP_X_UNICORNS`? Since they have their own Rack::Attack config blocks not relying on nginx details, I think we should keep them separate, like:

```
RATE_LIMIT_NUMBER: 300
RATE_LIMIT_PERIOD: 300
RATE_LIMIT_LOGIN_ATTEMPTS: 5
RATE_LIMIT_LOGIN_PERIOD: 20
RATE_LIMIT_PER_NGINX_UPSTREAM:
  [etc.]
```

Reply:
I agree, it also creates two additional blocks.